### PR TITLE
Webserver ui v1

### PR DESCRIPTION
**Pull request by alexthebake**
This pr is too big.. Anyway, I wanted to get a bit further before I pushed this up, but I ran into some issues, so I thought, what the hell, let's just make a pr. The next steps are to implement:

- Activating land
- Attacking and blocking with creatures
- Pretty UI

### CODE REVIEW

File: clients/webserver/deckr/webclient/static/js/game/Card.jsx [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+    if (this.props.zone == "hand") {
+      return this._handActions();
+    } else {
+      return this._battlefieldActions();
+    }
+  },
+  _handActions: function () {
+    return ["Play"];
+  },
+  _battlefieldActions: function () {
```
Comment (alexthebake) [inline clients/webserver/deckr/webclient/static/js/game/Card.jsx]: This should be loaded from the card data. The only issue is that the deckr server wasn't giving me anything but the card's name (I think the update_proto method for cards only changes the name)

Comment (courageousillumination) [inline clients/webserver/deckr/webclient/static/js/game/Card.jsx]: Yeah that's all it does right now (and the controller, and tapped status as of the latest commit). I've been slowly expanding the functionality with various commits. If there's things you really want let me know and i'll add them in.

File: clients/webserver/deckr/webclient/deck_builder.py [hunk 2]
```diff
@@ -0,0 +1,6 @@
+def build_deck(deck_text):
+    deck = []
+    for line in deck_text.split('\n'):
+        card = line.split(' ')
+        deck += [str(card[1]) for _ in range(int(card[0]))]
```
Comment (courageousillumination) [inline clients/webserver/deckr/webclient/deck_builder.py]: Lol. Let's figure out a better way to do this (it'll break on names that are more than one word). Fine for now though.

**Comment by courageousillumination** [comment]
Looks good to me. As always, I'd say tests. I'd advocate for next steps to focus on game elements over UI. That way I can use this to debug the game engine.

**Comment by alexthebake** [comment]
Yea good point. Okay! I implemented some tests (although I'm sure there are few more I could do). Right now the UI isn't super pretty, but it should give a lot of information. If you want it display more info, let me know.

**Pull request merged**